pallets/subtensor/src/utils/misc.rs

```diff
@@ -106,8 +109,16 @@ impl<T: Config> Pallet<T> {
     // ==================================
     // ==== YumaConsensus UID params ====
     // ==================================
+    pub fn set_last_crv3_update_for_uid(netuid: u16, uid: u16, last_update: u64) {
+        let mut updated_last_update_vec = Self::get_last_crv3_update(netuid);
+        let Some(updated_last_update) = updated_last_update_vec.get_mut(uid as usize) else {
+            return;
+        };
+        *updated_last_update = last_update;
+        LastCRV3Update::<T>::insert(netuid, updated_last_update_vec);
+    }
     pub fn set_last_update_for_uid(netuid: u16, uid: u16, last_update: u64) {
-        let mut updated_last_update_vec = Self::get_last_update(netuid);
+        let mut updated_last_update_vec = Self::get_last_crv3_update(netuid);
```

Comment:
A separate `get_last_update` tracker is not needed because `get_last_update` is already used throughout the codebase to track the last time weights were updated, regardless of the method, hence the failing childkey/emission tests.